include/psp2/json.h

```diff
@@ -276,7 +276,7 @@ class Array
 	Array& operator=(const Array& arr);
 
 private:
-	void* impl_data; //!< Pointer to internal implementation data
+	void* impl_data; //!< Pointer to internal implementation data.
```

Comment:
userData and errorCode are `camelCase` but `impl_data` is `snake_case`.
could you choose one notation for this lib?

Reply:
Done.